lib/admin/components/UserAdmin.js

```diff
@@ -66,30 +83,104 @@ export default class UserAdmin extends Component<Props> {
 
     // load orgs because they're needed both in org and user creation
     fetchOrganizations()
+
+    // load servers if deployments are enabled.
+    if (isModuleEnabled('deployment')) fetchServers()
   }
 
-  render () {
+  _getMainContent = (isApplicationAdmin: boolean): React.Node => {
     const {
       activeComponent,
       createUser,
       deleteUser,
       fetchProjectFeeds,
       fetchUsers,
       organizations,
+      otpServers,
       projects,
       setUserPage,
       setUserQueryString,
       updateUserData,
       user,
       users
     } = this.props
+    switch (activeComponent) {
+      case 'users':
+        if (!users.data || !projects || !organizations.data) {
+          return <Loading />
+        } else {
+          return (
+            <UserList
+              createUser={createUser}
+              creatingUser={user}
+              deleteUser={deleteUser}
+              fetchProjectFeeds={fetchProjectFeeds}
+              fetchUsers={fetchUsers}
+              isFetching={users.isFetching}
+              organizations={organizations.data}
+              page={users.page}
+              perPage={users.perPage}
+              projects={projects}
+              setUserPage={setUserPage}
+              setUserQueryString={setUserQueryString}
+              token={user.token || 'token-invalid'}
+              updateUserData={updateUserData}
+              userCount={users.userCount}
+              users={users.data}
+            />
+          )
+        }
+      case 'organizations':
+        if (!isApplicationAdmin || isModuleEnabled('enterprise')) return null
+        else return <OrganizationList {...this.props} />
+      case 'logs':
+        return <p className='text-center' style={{marginTop: '100px'}}>
+          <Button
+            bsStyle='danger'
+            bsSize='large'
+            href='https://manage.auth0.com/#/logs'>
+            <Icon type='star' /> View application logs on Auth0.com
+          </Button>
+        </p>
+      case 'servers':
+        if (!isApplicationAdmin || !isModuleEnabled('deployment')) {
+          return <p className='text-center lead'>Restricted access</p>
+        }
+        return <ServerSettings
```

Comment:
Not too many props on this one, but should probably also be a connected component.